test/helpers/db_helper.rb

```diff
@@ -112,11 +119,18 @@ def seed_simple_database_with_single_table
     seed_random_data(source_db, number_of_rows: max_id)
 
     # Create some holes in the data.
-    delete_statement = source_db.prepare("DELETE FROM #{full_table_name(DEFAULT_DB, DEFAULT_TABLE)} WHERE id = ?")
-    140.times do
-      delete_statement.execute(Random.rand(max_id))
+    num_holes = 140
```

Comment:
took me a while to figure out why this was named `num_holes` until I finally saw `# Create some holes in the data.` above 😆